### PR TITLE
Logging module.

### PR DESCRIPTION
**Pull request by timlesallen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,44 @@
 > Log json data to amqp from express routes.
```
Comment (eugeneware) [inline README.md]: If we've made this less about express, and more about logging generic data to amqp then there's still a few references to express in the README, etc.

File: package.json [hunk 2]
```diff
@@ -12,16 +12,16 @@
...
-    "url": "git://github.com/noblesamurai/express-amqp-logger.git"
+    "url": "git://github.com/noblesamurai/node-log2amqp.git"
   },
   "bugs": {
-    "url": "https://github.com/noblesamurai/express-amqp-logger/issues"
+    "url": "https://github.com/noblesamurai/node-log2amqp/issues"
   },
   "engines": {
-    "node": "6.x",
+    "node": ">=0.4.0 <8",
```
Comment (eugeneware) [inline package.json]: Does this break in 8?

Comment (eugeneware) [inline package.json]: Actually, I think you mean `>=4` as you use es2015 features like `let` which won't work in node `0.4.0`.

File: README.md (hunk 1 above)
Comment (timlesallen) [inline README.md]: Oops, good call.

File: package.json (hunk 2 above)
Comment (timlesallen) [inline package.json]: Yeah, thanks.  fixed.

**Pull request merged**